### PR TITLE
docs: FOUNDENG-439 Provide documentation for the proxy support we added to the launcher in FOUNDENG-425

### PR DESCRIPTION
**Pull request by rcorujo**
## Description

In FOUNDENG-425, we added support to the launcher to import the proxy variables "http_proxy", "https_proxy", "ftp_proxy", "rsync_proxy", "gopher_proxy", "socks_proxy", "socks5_server", and "no_proxy" from "/etc/environment" and "/etc/sysconfig/proxy" (SLES) and export them in lowercase and uppercase to the container.

This is the user documentation describing this proxy support.

In "Install Determined on Slurm/PBS", we added bullet item #7.

image

In "Installation Requirements", we added the following "Proxy Configuration Requirements" section.

image

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by phillip-gaisford — APPROVED** [review]
LGTM I do see some test failures though

**Comment by rcorujo** [comment]
@stoksc If I run a "make build" locally, the "webui/react/build" directory gets populated.  It appears that the commands being executed as part of the CircleCi workflow are not creating "webui/react/build"?

```
cp -r ../webui/react/build/* build/webui/react
cp: cannot stat '../webui/react/build/*': No such file or directory
```

This is the same error I got last time I had an OSS pull request.

**Comment by stoksc** [comment]
i thought the error last time was growforest not being installed? anyway, ill try something.

**Comment by rcorujo** [comment]
> i thought the error last time was growforest not being installed? anyway, ill try something.

This was the error last time.

```
goreleaser --snapshot --rm-dist
   • releasing...     
   • loading config file       file=.goreleaser.yml
   • loading environment variables
   • getting and validating git state
      • couldn't find any tags before "0.19.9-dev0"
      • building...               commit=3abd5057ba68d83abc2c60ceb98d52daa8eb0be8 latest tag=0.19.9-dev0
      • pipe skipped              error=disabled during snapshot mode
   • parsing tag      
   • running before hooks
      • running                   hook=make pre-package
   ⨯ release failed after 0.99s error=hook failed: make pre-package: exit status 2; output: make[2]: Entering directory '/home/circleci/project/master'
no required module provides package github.com/ryanbressler/CloudForest/growforest; to add it:
	go get github.com/ryanbressler/CloudForest/growforest
rm -rf build
mkdir -p build/webui/docs
mkdir -p build/webui/react
mkdir -p build/wheels/
cp -r ../proto/build/swagger build/swagger
cp -r ../docs/site/html/* build/webui/docs
cp -r ../webui/react/build/* build/webui/react
cp: cannot stat '../webui/react/build/*': No such file or directory
make[2]: *** [Makefile:108: pre-package] Error 1
make[2]: Leaving directory '/home/circleci/project/master'

make[1]: *** [Makefile:117: package] Error 1
make[1]: Leaving directory '/home/circleci/project/master'
make: *** [Makefile:28: package] Error 2
```

This is the error I'm getting now.  Seems the same.

```
no required module provides package github.com/ryanbressler/CloudForest/growforest; to add it:
	go get github.com/ryanbressler/CloudForest/growforest
rm -rf build
mkdir -p build/webui/docs
mkdir -p build/webui/react
mkdir -p build/wheels/
cp -r ../proto/build/swagger build/swagger
cp -r ../docs/site/html/* build/webui/docs
cp -r ../webui/react/build/* build/webui/react
cp: cannot stat '../webui/react/build/*': No such file or directory
make[2]: *** [Makefile:137: pre-package] Error 1
```

File: docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst [hunk 1]
```diff
@@ -54,6 +54,24 @@ The launcher has the following additional requirements on the installation node:
...
+
+**********************************
+ Proxy Configuration Requirements
+**********************************
+
+If internet connectivity requires a use of a proxy, verify the following requirements:
+
+-  Ensure that the proxy variables are defined in `/etc/environment` (or `/etc/sysconfig/proxy` on
+   SLES).
+-  Ensure that login, admin, and compute nodes, which resolve to short names, are included in the
```
Comment (jerryharrow) [inline docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst:67]: This sentence is difficult to parse.    Also it seems untenable to list all the compute nodes in the no_proxy list -- we don't define them on our test systems, so Determined apparently doesn't communicate between the compute nodes?   I would minimally separate out login/admin to be more prescriptive, and then add compute nodes separately (if we need to do so).

- Ensure that the `no_proxy` setting covers the login and admin nodes.   If these nodes may be referenced by short names known only within the cluster, they must explicitly be included in the `no_proxy` setting.
- If your experiment code communicates between compute nodes with a protocol that honors proxy environment variables, you should additionally include the names of all compute nodes in the `no_proxy` variable setting.

**Review by jerryharrow — COMMENTED** [review]
Good, but perhaps a bit more clarity on one bullet?

**Review by jerryharrow — APPROVED** [review]

**Comment by rcorujo** [comment]
Closing this pull request, as it keeps failing.

Replaced it with https://github.com/determined-ai/determined/pull/5934 from a forked repo.

**Pull request closed without merging**